vscode-build/vscode.patch

```diff
@@ -982,7 +999,7 @@ index 1fe5aa621c..18d257fab3 100644
  
 diff --git a/src/vs/workbench/browser/parts/dummies/activitybarPart.ts b/src/vs/workbench/browser/parts/dummies/activitybarPart.ts
 new file mode 100644
-index 0000000000..455a50bb45
```

Comment:
Why did all these other files change?

Reply:
it seems like they did not... only their hash became one charger longer...